CMakeLists.txt

```diff
@@ -170,7 +171,7 @@ if(NOT UNIX)
 endif(NOT UNIX)
 
 # Check if the external packages are installed into a separate install directory
-check_external_package_install_dir()
+#check_external_package_install_dir() # does not work with FR 19.0
```

Comment:
Has FairRoot 19.0 already been out? I checked their GitHub page and the latest release is still 18.6.9. Where did you get the 19.0 version?

Reply:
@klenze Are the changes to `CMakeLists.txt` necessary for the CALIFA overflow check?  Otherwise, could they be put in a separate PR, such that this PR can get merged?

Reply:
@YanzhaoW No, I think I used the dev version for FR.

I have removed the changes to CMakeLists.txt.